ansible/roles/sysprep/tasks/main.yml

```diff
@@ -205,3 +205,11 @@
 - name: Disable Swap
   command: swapoff -a
   when: swapon.stdout
+
+- name: Stop and disable firewalld service
+  systemd:
+    name: firewalld
+    enabled: false
+    masked: false
+    state: stopped
+  when: disable_firewalld
```

Comment:
Do we need?
```suggestion
  when: disable_firewalld | default(false)
```

Reply:
Does this make `disable_firewalld` default to `true`? Sorry I'm not that versed in ansible.